runtime/jcl/exports.cmake

```diff
@@ -718,3 +718,10 @@ if(NOT JAVA_SPEC_VERSION LESS 20)
 		Java_java_lang_Thread_setScopedValueCache
 	)
 endif()
+
+# J9VM_OPT_JFR
+if(J9VM_OPT_JFR)
+	omr_add_exports(jclse
+		Java_com_ibm_oti_vm_VM_triggerExecutionSample
+	)
+endif()
```

Comment:
Please move this up after J9VM_OPT_CRIU_SUPPORT so it's together with other "features".